packages/hooks/src/__tests__/useTotalAccountBalance.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useTotalAccountBalance from '../useTotalAccountBalance';
+
+describe('useTotalAccountBalance', () => {
+    test('should return zero when user has no account', async () => {
+        const mock = mockStore({
+            client: {
+                active_accounts: [],
+            },
+        });
```

Comment:
```suggestion
        const mock = mockStore({});
```
@shayan-deriv By default it's empty 🙇🏻
It's better to rely on the default values from the `mockStore` when we don't need to change it, It kinda helps us to have a test for the `mockStore` itself as if someone changes some default value to something else the test cases relying on those default values will fail, But if we override them then nothing happens.